Readability.js

```diff
@@ -724,8 +734,12 @@ Readability.prototype = {
         // Add the score to the parent. The grandparent gets half.
         parentNode.readability.contentScore += contentScore;
 
-        if (grandParentNode)
+        if (grandParentNode && grandParentNode.readability) {
```

Comment:
Maybe if `typeof(grandParentNode.tagName)` is undefined... that'll not initialize readability for that node...